lib/rpmi_service_group_syssusp.c

```diff
@@ -244,6 +244,8 @@ rpmi_service_group_syssusp_create(struct rpmi_hsm *hsm,
 	group = &sgsusp->group;
 	group->name = "syssusp";
 	group->servicegroup_id = RPMI_SRVGRP_SYSTEM_SUSPEND;
+	group->servicegroup_version =
+		RPMI_BASE_VERSION(RPMI_SPEC_VERSION_MAJOR, RPMI_SPEC_VERSION_MINOR);
```

Comment:
For now, service group version is same with RPMI version.
But, each service group should have its version (their version macros)?

Reply:
Yes, when they will deviate we can define the new MAJOR and MINOR macros

Reply:
Okay

Reply:
Shall we merge it then?